src/Illuminate/Cache/NullStore.php

```diff
@@ -25,6 +25,19 @@ public function get($key)
     }
 
     /**
+     * Retrieve multiple items from the cache by key.
+     *
+     * Items not found in the cache will have a null value for the key.
+     *
+     * @param  array  $keys
+     * @return array
+     */
+    public function getMultiple(array $keys)
+    {
+        //
```

Comment:
Updated that.  That method may have been more appropriate as a trait for the implementations that don't have a better way of handling getting/settings multiple values at once.